app/views/settings_dialog.py

```diff
@@ -686,6 +688,51 @@ def _do_todds_tab(self) -> None:
         )
         group_layout.addWidget(self.todds_overwrite_checkbox)
 
+    def _do_themes_tab(self) -> None:
+        tab = QWidget()
+        self.tab_widget.addTab(tab, "Theme")
+
+        tab_layout = QVBoxLayout(tab)
+        tab_layout.setAlignment(Qt.AlignmentFlag.AlignTop)
+
+        theme_group_box = QGroupBox("Themes")  # Added a title for the group box
```

Comment:
Doesn't have the correct styling compared to other group boxes

Reply:
ok done